paf-mvp-cmp/src/log.ts

```diff
@@ -0,0 +1,29 @@
+// Wrappers to console.(log | info | warn | error). Takes N arguments, the same as the native methods
+export class log {
```

Comment:
The same exists in https://github.com/prebid/paf-mvp-implementation/blob/main/paf-mvp-frontend/src/lib/logging.ts, should be refactored and put in common

Reply:
Agree. It was copied from there pending discussion.

Reply:
This refactoring has been added to this PR.